Framework/DataObjects/test/WorkspaceCreationTest.h

```diff
@@ -387,6 +408,31 @@ class WorkspaceCreationTest : public CxxTest::TestSuite {
     check_zeroed_data(*ws);
   }
 
+  void test_default_StorageMode_is_Cloned() {
+    IndexInfo indices(2);
+    indices.setSpectrumDefinitions(std::vector<SpectrumDefinition>(2));
+    TS_ASSERT_EQUALS(
+        create<Workspace2D>(indices, BinEdges{1, 2, 4})->storageMode(),
```

Comment:
Good. I think this is important to check, particularly if someone starts changing what the defaults are on the Workspace subtypes.